org.opent2t.sample.lamp.superpopular/com.wink.lightbulb/js/tests/test.js

```diff
@@ -3,66 +3,84 @@ var test = require('ava');
 var OpenT2T = require('opent2t').OpenT2T;
 var config = require('./testConfig');
 
-console.log('Device Under Test -  Name: ' + config.Device.name + '  Props: ' + JSON.stringify(config.Device.props));
+var config = require('./testConfig');

```

Comment:
Good catch. Removed.

---
In reply to: [78666577](https://github.com/openT2T/translators/pull/95#discussion_r78666577) [](ancestors = 78666577)